README.md

```diff
@@ -1,29 +1,286 @@
-# prototype-ci-reading
+# Proposed Quality Workflow (QW) user guide - release 1
 
-Quick prototype for processing PR and issue data in github actions
+The Quality Workflow tool (QW) aims to help reduce the friction in creating and maintaining traceability matrices (later release could target hazards and risks).
 
-## Ensuring that we can access these parts in CI
+# Background and use-case
 
-Issues
+Within medical device software there are several design and development stages that should be tracked, versioned and a traceability matrix is a standard way of presenting this information. The Quality Management System used in WEISS is document based, which requires substantial manual effort in tracking linkages, updating of versions and ensuring that all approvals are current.
 
-- [x] Title
-- [x] First comment
-- [x] PR number
-- [x] Assignee(s)
-  - If we're using names, they're not required on github so could end up being blank
-  - We could just use github logins and have part of the config that you have to map gh logins
-    to names as you want them on the report information
-- [x] Label
+The QW allows users to track the different changes in design and development requirements, outputs, verification and validations. These different stages are linked together using github or gitlab as part of the normal development process. QW also will ensure that versioning of items are updated if their contents have changed, and notify when manual verification should be rerun due to changes in requirements or outputs.
 
-PRs
+# Setup
 
-- [x] Title
-- [x] First comment
-- [x] Assignee(s)
-- [x] Users who's final reviews were APPROVED
-- [x] Label
-- [x] [Linked Issues](https://github.com/cli/cli/discussions/7097#discussioncomment-5229031)
+## Installation
 
-## Prototyping of QW tool
+The `qw` tool requires python 3.9 or greater, if your project is in python then we suggest adding it to the developer requirements. Otherwise may want to create a virtual environment for your python, one option is the [venv](https://docs.python.org/3.9/library/venv.html) module.
```

Comment:
Otherwise *you may

Reply:
```suggestion
The `qw` tool requires python 3.9 or greater, if your project is in python then we suggest adding it to the developer requirements. Otherwise you may want to create a virtual environment for your python, one option is the [venv](https://docs.python.org/3.9/library/venv.html) module.
```